packages/webex/jest.config.js

```diff
@@ -1,3 +1,6 @@
-import config from '@webex/jest-config-legacy';
+const config = require('@webex/jest-config-legacy');
 
-export default config;
+module.exports = {
+  ...config,
+  testEnvironment: 'jsdom',
```

Comment:
<img width="1000" alt="Screenshot 2024-02-29 at 12 41 49 PM" src="https://github.com/webex/webex-js-sdk/assets/72344404/512f4c12-2de0-4b83-8859-99fc455b6a7e">
We are trying to access the window object in one of the tests, thats why this change was required.

Reply:
I think this is because something inside the internal-media-core lib is trying to access the window object. The better thing to do here is to mock the window object and the method that's required, rather than execute the test in jsdom.

Reply:
Yes, the error is happening because of internal-media-core is accessing windows. Have mocked plugin-meetings packages itself.